internal/theia/cbom.go

```diff
@@ -0,0 +1,47 @@
+package theia
```

Comment:
It should not be `theia`, this is different project owned by PQCA, we are building open source Seeker tool.